src/Aspire.Hosting.Azure/Bicep/signalr.bicep

```diff
@@ -0,0 +1,81 @@
+param name string
+param principalId string
+
+@description('Tags that will be applied to all resources')
+param tags object = {}
+
+@description('The pricing tier of the SignalR resource.')
+@allowed([
+  'Free_F1'
+  'Standard_S1'
+  'Premium_P1'
+])
+
+param pricingTier string = 'Free_F1'
```

Comment:
should we call this sku?

Reply:
sku contains both unit number and pricing tier  
![image](https://github.com/dotnet/aspire/assets/668244/25b049fe-0c73-4230-850a-c741734543d9)

Reply:
Got it.